### ISSUE TITLE
Different installation location

### ISSUE THREAD

**Issue by Nauman3S** [litespeedtech/cyberpanel#1043]
**Describe the bug**

I have a Ubuntu 22.04 VPS with /root partition of only 25GB(which is not enough for my websites) and extra storage of 300GB mounted at `/mnt/ext_storage/`. 
The issue is CyberPanel installs everything in `/usr/local` directory and doesn't have any storage directory selection procedure during the installation. I moved the `/usr/local/Cyber*` directories to `/mnt/ext_storage/local/` and created a symlink to the moved files at `/usr/local/` to circumvent the issue but the CyberPanel now doesn't fully load.

**To Reproduce**

move the `/usr/local/Cyber*` directories to `/mnt/ext_storage/local` or any other directory and create a symlink to the moved files at `/usr/local/`

**Expected behavior**

CyberPanel is not loading at all.

**Screenshots**
![im](https://user-images.githubusercontent.com/8595825/229115913-ab6af991-28f2-482e-ba27-3bf03945ac0c.png)

**Operating system:**
Ubuntu 22.04 LTS

**CyberPanel version:**
latest

**Comment by usmannasir** [comment]
How about mounting before installation, does it still not work?

**Comment by Nauman3S** [comment]
> How about mounting before installation, does it still not work?

No, it will not work as I can't specify the installation location. It's always `/usr/local`

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
